q2cli/builtin/tools.py

```diff
@@ -65,6 +65,10 @@ def export_data(input_path, output_path, output_format):
                 if os.path.isfile(output_path):
                     os.remove(output_path)
                 else:
+                    # This allows the user to pass a filename as a path if they
+                    # want their output in the current working directory
+                    if os.path.dirname(output_path) == '':
+                        output_path = os.path.join('.', output_path)
```

Comment:
Instead of adding work to be undone, what if we only ran `os.makedirs` when there _is_ a directory in the path. Otherwise, it must be a filename and the duplicate call will take care of that fine.